### ISSUE TITLE
Use digitalSpecimenId as link when uploading media objects

### ISSUE THREAD

**Issue by jgrieb** [DiSSCo/dissco-core-digital-media-object-processor#6]
When upserting a new DigitalMedia object via POST currently the (required) ID of the DS the media belongs to has to be given in the JSON via 
```
"physicalSpecimenId": "https://id.senckenberg.de/object/objectxyz",
"idType": "cetaf",
```
I would propose to change this to passing the actual ID of the digital specimen object
`"digitalSpecimenId": "20.5000.1025/IDXYZ,"`
Then it would also be more in line with how a DigitalMedia objects are retrieved via GET

**Comment by samleeflang** [comment]
Yes good point. Now it assumes that all the data comes from CMS's which don't have the handles yet. So we do the lookup for the handle of the associated digitalSpecimen based on the physical Specimen Id. This can change when the actual handle is directly provided as in the case of OpenRefine. We will put this on the backlog and when we have some time left (probably not in Oct or Nov) we will implement this.
